### PR TITLE
BZ1323272 - DB: Process Instances history for process variables does …

### PR DESCRIPTION
**Pull request by nmirasch**
…not work for all Oracle DB

### CODE REVIEW

File: jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/test/java/org/jbpm/console/ng/pr/client/editors/variables/history/VariableHistoryPopupTest.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        ProcessVariableSummary processVariableSummary2 = new ProcessVariableSummary();
+        ProcessVariableSummary processVariableSummary3 = new ProcessVariableSummary();
+
+        processVariableSummary1.setNewValue("A");
+        processVariableSummary2.setNewValue("B");
+        processVariableSummary3.setNewValue("A");
+
+        Comparator varNewValueComparator = variableHistoryPopup.getSortHandler().getComparator(testDataGrid.getColumn(COLUMN_NEW_VALUE_POSITION));
+
+        assertEquals(varNewValueComparator.compare(processVariableSummary1, processVariableSummary2), -1);
```
Comment (cristianonicolai) [inline jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/test/java/org/jbpm/console/ng/pr/client/editors/variables/history/VariableHistoryPopupTest.java]: expected value should be the first parameter to assertEquals, please review statements.

File: jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/variables/history/VariableHistoryPopup.java [hunk 2]
```diff
@@ -123,7 +133,7 @@ public void onRangeChange( RangeChangeEvent event ) {
 
             @Override
             public String getValue( ProcessVariableSummary object ) {
-                return object.getNewValue();
+                return (object.getNewValue()!=null? object.getNewValue():"") ;
```
Comment (cristianonicolai) [inline jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/variables/history/VariableHistoryPopup.java]: Please review formatting and remove the extra parenthesis as it is not needed.

**Comment by cristianonicolai** [comment]
@nmirasch looks good, just some minor updates needed.

**Comment by nmirasch** [comment]
@cristianonicolai thanks, done the updates!

**Comment by cristianonicolai** [comment]
+1

**Pull request merged**